app/src/main/java/com/monkeyteam/chimpagne/model/database/ChimpagneRoles.kt

```diff
@@ -1,8 +1,9 @@
 package com.monkeyteam.chimpagne.model.database
 
 typealias ChimpagneRole = Int
+
 object ChimpagneRoles {
-  val OWNER : ChimpagneRole = 0
-  val STAFF : ChimpagneRole = 1
-  val GUEST : ChimpagneRole = 2
-}
\ No newline at end of file
+  val OWNER: ChimpagneRole = 0
```

Comment:
An enum would be more approriate for this

Reply:
I'm unsure if enum are Firebase friendly, but we can give it a try

Reply:
This was not made in this branch, I am merely using it so if you want to change it, it will be done in another PR.